File: C/10.c
#include <stdio.h>
int is_prime(int num);
int main()
{
    long double number = 0;
    long double sum = 0;

    while (number < 2000000)
    {
        number++;
        if (is_prime(number))
        {
            sum += number;
        }
    }
    printf("%Lf\n", sum-1);
}

int is_prime(int num)
{
    int i;
    for (i=2; i<num/2+1; i++)
    {
        if (num % i == 0)
        {
            return 0;
        }
    }
    return 1;
}

File: python/10.py
sum = 0
num = 0

def is_prime(num):
    for i in xrange(2,num/2+1):
        if num % i == 0:
            return False
    return True

while num < 2000000:
    num += 1
    if is_prime(num): 
	sum += num

print sum-1
